mappings/net/minecraft/client/gui/screen/world/EditGameRulesScreen.mapping

```diff
@@ -19,70 +19,71 @@ CLASS net/minecraft/unmapped/C_ajacvfls net/minecraft/client/gui/screen/world/Ed
-			ARG 1 name
-			ARG 2 description
-			ARG 3 ruleName
-			ARG 4 rule
-	CLASS C_lppnmpti NamedRuleWidget
-		FIELD f_ggwwaclx name Ljava/util/List;
+			ARG 1 message
+			ARG 2 tooltip
+			ARG 3 description
+			ARG 4 gameRule
+	CLASS C_lppnmpti GameRuleEntry
+		FIELD f_ggwwaclx message Ljava/util/List;
 		FIELD f_nxcbmpxi children Ljava/util/List;
 		METHOD <init> (Lnet/minecraft/unmapped/C_ajacvfls;Ljava/util/List;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
-			ARG 2 description
-			ARG 3 name
-		METHOD m_getgtnya drawName (Lnet/minecraft/unmapped/C_sedilmty;II)V
-			ARG 2 x
-			ARG 3 y
-	CLASS C_nulgdlho RuleCategoryWidget
-		FIELD f_oqcsioxt name Lnet/minecraft/unmapped/C_rdaqiwdt;
+			ARG 2 tooltip
+			ARG 3 message
+		METHOD m_getgtnya drawMessage (Lnet/minecraft/unmapped/C_sedilmty;II)V
+			ARG 1 graphics
+			ARG 2 y
+			ARG 3 x
+	CLASS C_nulgdlho CategoryEntry
+		FIELD f_oqcsioxt title Lnet/minecraft/unmapped/C_rdaqiwdt;
 		METHOD <init> (Lnet/minecraft/unmapped/C_ajacvfls;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
-			ARG 2 text
-	CLASS C_vbuitpof IntRuleWidget
-		FIELD f_rrnlapbl valueWidget Lnet/minecraft/unmapped/C_vopzcnhf;
+			ARG 2 title
+	CLASS C_vbuitpof IntGameRuleEntry
+		FIELD f_rrnlapbl valueField Lnet/minecraft/unmapped/C_vopzcnhf;
 		METHOD <init> (Lnet/minecraft/unmapped/C_ajacvfls;Lnet/minecraft/unmapped/C_rdaqiwdt;Ljava/util/List;Ljava/lang/String;Lnet/minecraft/unmapped/C_xmldumst$C_tdajhqee;)V
-			ARG 2 name
-			ARG 3 description
-			ARG 4 ruleName
-			ARG 5 rule
+			ARG 2 message
+			ARG 3 tooltip
+			ARG 4 description
+			ARG 5 gameRule
 		METHOD m_yamhjwfu (Lnet/minecraft/unmapped/C_xmldumst$C_tdajhqee;Ljava/lang/String;)V
 			ARG 2 value
-	CLASS C_vdrulthz AbstractRuleWidget
-		FIELD f_eexekwhn description Ljava/util/List;
+	CLASS C_vdrulthz AbstractEntry
+		FIELD f_eexekwhn tooltip Ljava/util/List;
 		METHOD <init> (Ljava/util/List;)V
-			ARG 1 description
-	CLASS C_xpragfeh BooleanRuleWidget
+			ARG 1 tooltip
+	CLASS C_xpragfeh BooleanGameRuleEntry
 		FIELD f_ofxfnaaa toggleButton Lnet/minecraft/unmapped/C_ikfvpkkf;
 		METHOD <init> (Lnet/minecraft/unmapped/C_ajacvfls;Lnet/minecraft/unmapped/C_rdaqiwdt;Ljava/util/List;Ljava/lang/String;Lnet/minecraft/unmapped/C_xmldumst$C_juzzykai;)V
-			ARG 2 name
-			ARG 3 description
-			ARG 4 ruleName
-			ARG 5 rule
+			ARG 2 message
+			ARG 3 tooltip
+			ARG 4 description
+			ARG 5 gameRule
 		METHOD m_wzrvbdmo (Lnet/minecraft/unmapped/C_xmldumst$C_juzzykai;Lnet/minecraft/unmapped/C_ikfvpkkf;Ljava/lang/Boolean;)V
 			ARG 1 button
 			ARG 2 value
 		METHOD m_znouopbs (Ljava/lang/String;Lnet/minecraft/unmapped/C_ikfvpkkf;)Lnet/minecraft/unmapped/C_npqneive;
 			ARG 1 button
-	CLASS C_ymfbyxcw RuleListWidget
+	CLASS C_ymfbyxcw GameRuleEntryListWidget
 		METHOD <init> (Lnet/minecraft/unmapped/C_ajacvfls;Lnet/minecraft/unmapped/C_xmldumst;)V
 			ARG 2 gameRules
 		METHOD m_evugyeiw (Ljava/util/Map$Entry;)V
-			ARG 1 entry
+			ARG 1 categoryMapEntry
```

Comment:
I think the `map` this is for should also be renamed to `categoryMap`.

Reply:
It can't be because it's a local.

Reply:
Oh locals can't be mapped... shux

Reply:
I'm like 90% sure they can't.  If they can... I sure have some mapping to do, because it would make my own mod code a lot easier to understand if the locals within `createGameRuleEntry` in these mappings were mapped, which is defined just below where this map is defined actually.

Resolving for now unless someone has anything to add on locals (not) being mappable.